game/scripts/vscripts/TutorialGraph/Steps.ts

```diff
@@ -14,13 +14,21 @@ export const goToLocation = (location: tg.StepArgument<Vector>, visualIntermedia
     let checkTimer: string | undefined = undefined
     let pathParticle: ParticleID | undefined = undefined
     let actualLocation: Vector | undefined = undefined
+    let moveParticleFx: ParticleID | undefined = undefined
 
     const cleanup = () => {
         if (pathParticle) {
             ParticleManager.DestroyParticle(pathParticle, false)
+            ParticleManager.ReleaseParticleIndex(pathParticle)
```

Comment:
Do you still need this after destroy particle? If so, probably needed in more places that assume you don't

Reply:
Yes. Even when you destroy the particle, the system retains the index. You always have to release it.
The only exception is when doing AddParticle to attach to a modifier, which destroys and releases it for you.